### DISCUSSION TITLE
Create new subdirectory for subreddit/user in master folder?

### DISCUSSION THREAD

**Discussion by SFtheGreat** [Serene-Arc/bulk-downloader-for-reddit#838] [General]
Hello.
Total noob question, how to encode a creation of new directory for specified subreddit/user?

**Comment by Fakeaccount12312** [comment]
You do this using --folder-scheme.

For example --folder scheme "{SUBREDDIT}" to make folders per subreddit (the default behavior), --folder-scheme "{REDDITOR}" to make folders per user, --folder-scheme "{SUBREDDIT}/{REDDITOR}" to make folders per subreddit with subfolders per Redditor, and --folder-scheme "" to drop all files in the same folder.

https://github.com/aliparlakci/bulk-downloader-for-reddit#folder-and-file-name-schemes
